src/test/Test.FAKECore/ReleaseNotesSpecs.cs

```diff
@@ -132,7 +132,7 @@ public class when_parsing_many_pre_release_versions
 #### 2.0.0-alpha001 - December 15 2013
 * A";
         static readonly ReleaseNotesHelper.ReleaseNotes expected =
-            ReleaseNotesHelper.ReleaseNotes.New("2.0.0", "2.0.0-rc007", null, new[] { "A" }.ToFSharpList());
+            ReleaseNotesHelper.ReleaseNotes.New("2.0.0", "2.0.0-rc2", null, new[] { "A" }.ToFSharpList());
```

Comment:
Oh that's breaking in fake. Not sure if we really want that.

On Jul 24, 2016 6:56 PM, "Chester Husk III" <notifications@github.com>
wrote:

> In src/test/Test.FAKECore/ReleaseNotesSpecs.cs
> <https://github.com/fsharp/FAKE/pull/1325#discussion_r71991697>:
>
> > @@ -132,7 +132,7 @@ public class when_parsing_many_pre_release_versions
> >  #### 2.0.0-alpha001 - December 15 2013
> >  * A";
> >          static readonly ReleaseNotesHelper.ReleaseNotes expected =
> > -            ReleaseNotesHelper.ReleaseNotes.New("2.0.0", "2.0.0-rc007", null, new[] { "A" }.ToFSharpList());
> > +            ReleaseNotesHelper.ReleaseNotes.New("2.0.0", "2.0.0-rc2", null, new[] { "A" }.ToFSharpList());
>
> I made this change because the comparison logic in the spec changed. rc007
> is now less than rc2, so the expected output for the given release notes
> data had to change to take the improved comparison rules into account.
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/fsharp/FAKE/pull/1325/files/799b4be354b7eb8da836eec694fe7385d43411d7#r71991697>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AADgNHkwzbdlZ2itA60z_q8eYrUxUSdNks5qY5k1gaJpZM4JSHRJ>
> .
>